helm/cluster-api/templates/cert-manager.io_v1_certificate_capi-kubeadm-bootstrap-serving-cert.yaml

```diff
@@ -21,3 +21,6 @@ spec:
     kind: ClusterIssuer
     name: selfsigned-giantswarm
   secretName: capi-kubeadm-bootstrap-webhook-service-cert
+  subject:
```

Comment:
@giantswarm/team-phoenix are we fine with leaving this subject field like this in the certificate?